runtime/tests/test_cast.py

```diff
@@ -0,0 +1,50 @@
+import numpy as np
+import pytest
+from onnion_runtime import Cast
+
+from .utils import check, on_arm32
+
+
+def test_cast_00():
+    opset_version = 13
+    t = 1  # FLOAT
```

Comment:
What is the reason you don't use `onnx.TensorProto`, like legacy test?

Reply:
`onnx` is not installed in `idein/onnion-arm32-env`. 
[`python3-onnx`](https://packages.debian.org/bullseye/python3-onnx) is old (version 1.7) and the installation by pip is failed.

Reply:
Thanks, I got it.